config/config.go

```diff
@@ -66,14 +66,16 @@ anybar-port: {{.AnybarPort}}
 keen-write-key: {{.KeenWriteKey}}
 keen-project-id: {{.KeenProjectId}}
 keen-event-collection: {{.KeenEventCollection}}
-environment: {{.Namespace}}`)
+environment: {{.Namespace}}
+# Do not change the kubernetes-config-key. If it has been changed please run the setup command again
+kubernetes-config-key: {{.Namespace}}`)
 	err = tmpl.Execute(f, config)
 	return err == nil
 }
 
 //takes the application config from viper and checks that all the required fields are populated
 func Validate() (n int, missing []string) {
-	var mandatorySettings = [7]string{"project-key", "remote-branch", "remote-name", "cluster-ip", "username", "password", "kubernetes-config-key"}
+	var mandatorySettings = []string{"project-key", "remote-branch", "remote-name", "cluster-ip", "container", "username", "password", "kubernetes-config-key"}
```

Comment:
I see, yes that has been my mistake to add it while resolving the conflict. 

Although, since you said may be better this way I'll change the question to `"What is the default container for the watch, bash, fetch and resync commands?"` removing ` (Optional)`